### DISCUSSION TITLE
PSADT v4.0 fluent GUI display DeferDeadline

### DISCUSSION THREAD

**Discussion by philippjoos** [PSAppDeployToolkit/PSAppDeployToolkit#1106] [Q&A] [answered]
Hi everyone,

We are currently using version 3.10.1 to deploy our application with the -DeferDeadline parameter to set a deadline date and time. Our retry is configured with MECM, and users have the option to defer until the deadline is reached. Therefore, it is important for us to display the deadline date and time. I have attached a screenshot from version 3.10.1 for reference.
![psadt_deferdeadline](https://github.com/user-attachments/assets/421450a9-0e7e-405d-bf29-24ea5bfd022d)

I have now tried version 4.0 with the new Fluent GUI, but the deadline message was not displayed. From what I can see, there is no option in the new GUI to display the deadline in the same way as the message in the old GUI.

Am I doing something wrong, or is this option no longer available? Or is it simply not yet implemented?

Thanks and best regards,
Philipp

**Comment by mjr4077au** [comment, marked as answer]
I've transferred this to https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/issues/1111 since we need to track its reimplementation. Thanks for the report!
